src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -136,6 +134,26 @@ public function get_item_permissions_check( $request ) {
 		);
 	}
 
+	/**
+	 * Checks if a theme can be read.
+	 *
+	 * @since 5.5.0
+	 *
+	 * @return bool Whether the theme can be read.
+	 */
+	public function check_read_permission() {
```

Comment:
```suggestion
	protected function check_read_active_theme_permission() {
```

To make it more clear this check is just for the active theme, not any read.

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/264/commits/640bf1f74feae0833e2a850941ecce4aa5446b25